usage/README.rst

```diff
@@ -0,0 +1,77 @@
+Usage
+=====
```

Comment:
Most of this README was written by @mxk62 so I think it makes more sense to include his work in the previous commit, and only have your modifications in this commit.

Reply:
So why not include this commit with the previous one?

Reply:
I checked when I rebased and I thought that his original README was properly fixup'd into the first commit ("Add Mikolaj's code").  Apparently not. As I had mentioned in the comments of Jira, I had a huge issue with squashing the original README into the "Add Mikolaj's code" commit and apparently that is what happened here.  I'll keep trying to figure out how to get the original README into the "Add Mikolaj's code" commit.

Reply:
As you've already pushed to master we'll live with what it is.